src/Visualization/Vtk.jl

```diff
@@ -17,6 +17,40 @@ function createvtk(args...;kwargs...)
     visdata.grid,visdata.filebase,celldata=visdata.celldata,nodaldata=visdata.nodaldata)
 end
 
+"""
+"""
+function createpvd(args...;kwargs...)
+  paraview_collection(args...;kwargs...)
+end
+
+function createpvd(parts::Nothing,args...;kwargs...)
+  createpvd(args...;kwargs...)
+end
+
+function createpvd(f::Function,args...;kwargs...)
+  pvd = createpvd(args...;kwargs...)
+  try
+    f(pvd)
+  finally
+    savepvd(pvd)
+  end
+end
+
+function createpvd(f::Function,parts::Nothing,args...;kwargs...)
```

Comment:
I think that this one is redundant. Previous function already does the job for this case, right?

Reply:
Even better: keep this function but remove `::Function` so that one cas use other types of callable objects.

Reply:
I have quick-fixed this myself.